src/applications/mhv-landing-page/tests/medications/e2e/medications-landing-page-feature-toggle-false.cypress.spec.js

```diff
@@ -0,0 +1,48 @@
+import vamcEhr from '../../fixtures/vamc-ehr.json';
+
+const medicationFeatureToggles = (enabled = true) => ({
+  data: {
+    type: 'feature_toggles',
+    features: [
+      {
+        name: 'mhv_medications_to_va_gov_release',
+        value: enabled,
+      },
+    ],
+  },
+});
+
+describe('Medications Landing Page', () => {
+  beforeEach(() => {
+    cy.intercept('GET', '/data/cms/vamc-ehr.json', vamcEhr).as('vamcEhr');
+    const redirectPath = '/health-care/refill-track-prescriptions';
+    cy.intercept('GET', redirectPath, '').as('refillRedirect');
+  });
+
+  it('visits Medications landing Page', () => {
+    // Login when feature disabled, expect redirect
+    cy.intercept(
+      'GET',
+      '/v0/feature_toggles*',
+      medicationFeatureToggles(false),
+    );
+    cy.login();
+    cy.visit('my-health/about-medications/');
+
+    cy.wait('@refillRedirect');
+    cy.url().should('include', '/health-care/refill-track-prescriptions');
+
```

Comment:
Seems like the code below here is duplicate from what's in `/medications-landing-page.cypress.spec.js`.

I think lines 35 and on can safely be removed.

Reply:
I would want separate `it` blocks for each time `cy.login` is called, since that indicates we are dealing with different scenarios. It would be fine if both scenarios are in the same file.

Reply:
agreed, that would help with clarity reading it. 

// Utility function to setup feature toggles
const setupFeatureToggle = (enabled) => {
  cy.intercept('GET', '/v0/feature_toggles*', medicationFeatureToggles(enabled)).as('featureToggles');
};

describe('Medications Landing Page', () => {
  beforeEach(() => {
    cy.intercept('GET', '/data/cms/vamc-ehr.json', vamcEhr).as('vamcEhr');
    // Common setup for both tests
  });

  it('redirects when feature is disabled', () => {
    setupFeatureToggle(false);
    cy.login();
    cy.visit('my-health/about-medications/');
    cy.wait('@refillRedirect');
    cy.url().should('include', '/health-care/refill-track-prescriptions');
  });

  it('displays the regular page when feature is enabled', () => {
    setupFeatureToggle(true);
    cy.login();
    cy.visit('my-health/about-medications/');
    cy.injectAxe();
    cy.axeCheck('main', {
      rules: {
        'aria-required-children': { enabled: false },
      },
    });
  });
});

Reply:
As you wish